announcements/2021/week05.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This week in provider-independent announcements (February 05, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[External DNS v2.0.0](https://docs.giantswarm.io/changes/managed-apps/external-dns-app/v2.0.0/) allows customers to install and use External DNS for their own purpose, separate from the preinstalled app in AWS and Azure clusters. :warning: [Requires NGINX IC v1.14.0+](https://github.com/giantswarm/external-dns-app#limitations).
+
+[NGINX IC v1.14.0](https://docs.giantswarm.io/changes/managed-apps/nginx-ingress-controller-app/v1.14.0/) is released to work well with customer's separate External DNS (see above).
+
+[Cert Manager v2.4.2](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.4.2/) provides the option to automatically delete secrets when the parent Certificate is deleted, reducing unnecessary alerts.
+
+[K8s Initiator v0.9.2.](https://docs.giantswarm.io/changes/playground-apps/k8s-initiator-app/v0.9.2/) adds flexibility by adding support for user-specific tolerations and customer RBAC rules. We also added usage examples to increase user-friendliness. 
```

Comment:
@pipo02mix @QuentinBisson can you check if this is accurate?

Reply:
perfect!

Reply:
thanks for checking!